reviewed the docs functions to fix broken links and other additions (#14331)
netdata/netdata: docs/cloud/runtime-troubleshooting-with-functions.md

### Prerequisites

The following is required to be able to run Functions from Netdata Cloud.

> * Have at least one capable node claimed to one of your Spaces where node needs to be on a version higher than `v1.37`

Reviewer: ```suggestion
* At least one of the nodes claimed to your Space should be on a Netdata agent version higher than `v1.371.
```

Reviewer 2: a typo v1.371 => v1.37.1

Author: I'll take in @shyamvalsan 's suggestion and will take care of the typo. @ilyam8 
thanks for checking that, I thought the functions capability had been introduced on `v1.37`

Author: pushed a commit for this

> * Ensure that node has the collector that exposes the function you want enabled ([see current available functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#what-are-the-current-available-functions)

Reviewer: ```suggestion
* Ensure that the node has the collector that exposes the function you want enabled ([see current available functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#what-are-the-current-available-functions)
```

2. You are directed to the **Functions** tab
3. Follow the above instructions from step 3.

> ⚠️ If you get an error saying that your node can't execute Functions please check the [prerequisites](#prerequisites).

## Related Topics

### **Related Concepts**
- [Netdata Functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md)

#### Related References documentation

> - [External plugins overview](https://github.com/netdata/netdata/blob/master/docs/nightly/references/collectors-references/plugins.d/#function)

Reviewer 2: This link is 404

Author: seems @tkatsoulas made a suggestion will try that one

Author: fixed by putting the direction to the README.md file

> - [External plugins overview](https://github.com/netdata/netdata/blob/master/docs/nightly/references/collectors-references/plugins.d/#function)

Reviewer 3: ```suggestion
- [External plugins overview](https://github.com/netdata/netdata/tree/master/collectors/plugins.d#function)
```
